inference-engine/tests/functional/plugin/cpu/fusing_tests/src/group_convolution_fusing.cpp

```diff
@@ -0,0 +1,246 @@
+    groupConvLayerTestParamsSet basicParamsSet;
+    CPUSpecificParams cpuParams;
+    fusingSpecificParams fusingParams;
+    std::tie(basicParamsSet, cpuParams, fusingParams) = obj.param;
+
+    std::ostringstream result;
+    result << LayerTestsDefinitions::GroupConvolutionLayerTest::getTestCaseName(testing::TestParamInfo<groupConvLayerTestParamsSet>(
+            basicParamsSet, 0));
+
+    std::vector<cpu_memory_format_t> inFmts, outFmts;
+    std::vector<std::string> priority;
+    std::string selectedType;
+    std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
+
+    result << "_inFmts=" << CPUTestUtils::fmts2str(inFmts);
+    result << "_outFmts=" << CPUTestUtils::fmts2str(outFmts);
+    result << "_primitive=" << selectedType;
+
+    std::shared_ptr<ngraph::Function> postFunction;
+    std::vector<postNode> postNodes;
+    std::vector<std::string> fusedOps;
+    std::tie(postFunction, postNodes, fusedOps) = fusingParams;
+
+    if (postFunction) {
+        result << "_Fused=" << postFunction->get_friendly_name();
+    } else {
+        result << "_Fused=" << FusingTestUtils::postNodes2str(postNodes);
+    }
+
+    return result.str();
+}
+
+void GroupConvolutionLayerFusingTest::SetUp() {
+    groupConvLayerTestParamsSet basicParamsSet;
+    CPUSpecificParams cpuParams;
+    fusingSpecificParams fusingParams;
+    std::tie(basicParamsSet, cpuParams, fusingParams) = this->GetParam();
+
+    std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
+    std::tie(postFunction, postNodes, fusedOps) = fusingParams;
+
+    groupConvSpecificParams groupConvParams;
+    std::vector<size_t> inputShape;
+    auto netPrecision = InferenceEngine::Precision::UNSPECIFIED;
+    std::tie(groupConvParams, netPrecision, inputShape, targetDevice) = basicParamsSet;
+
+    ngraph::op::PadType padType;
+    InferenceEngine::SizeVector kernel, stride, dilation;
+    std::vector<ptrdiff_t> padBegin, padEnd;
+    size_t convOutChannels, numGroups;
+    std::tie(kernel, stride, padBegin, padEnd, dilation, convOutChannels, numGroups, padType) = groupConvParams;
+
+    auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+    ngraph::ParameterVector params = ngraph::builder::makeParams(ngPrc, {inputShape});
+    auto paramOuts = ngraph::helpers::convert2OutputVector(
+            ngraph::helpers::castOps2Nodes<ngraph::op::Parameter>(params));
+    auto groupConv = std::dynamic_pointer_cast<ngraph::opset1::GroupConvolution>(
+            ngraph::builder::makeGroupConvolution(paramOuts[0], ngPrc, kernel, stride, padBegin,
+                                                  padEnd, dilation, padType, convOutChannels, numGroups));
+    groupConv->get_rt_info() = CPUTestUtils::setCPUInfo(inFmts, outFmts, priority);
+
+    if (postFunction) {
+        function = makeNgraphFunction(ngPrc, params, groupConv, postFunction);
+    } else {
+        function = makeNgraphFunction(ngPrc, params, groupConv, postNodes);
```

Comment:
Why do you need such split of logic `if (postFunction)` .
From general point of view that's the same. Abstract ngraph function which will be attached after conv primitive.
Function/Node differentiation may be described in test param definition. Am I right?